### PR TITLE
kernel: update kernel 4.9 to 4.9.19

### PR DESCRIPTION
**Pull request by Ansuel**
Refresh mvebu patch , no other changes to generic patch

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

### CODE REVIEW

**Comment by Ansuel** [comment]
@wongsyrone i did just a target/linux/refresh... Don't know why :/ Should i remove them ?

**Comment by wongsyrone** [comment]
Well. Maybe I did refresh when adding support for .17

**Comment by Ansuel** [comment]
probably

**Comment by hnyman** [comment]
You should also refresh the patches in each target. Otherwise somebody else needs to do that cleanup...

There is even a script for it: https://gist.github.com/KanjiMonster/dfcd1af3190aa7b2941f1bdcb119f25e

**Comment by wongsyrone** [comment]
Although you said upgraded to 4.9.19, I cannot see the change in kernel-version.mk

File: include/kernel-version.mk [hunk 1]
```diff
@@ -4,11 +4,11 @@ LINUX_RELEASE?=1
 
 LINUX_VERSION-3.18 = .43
 LINUX_VERSION-4.4 = .56
-LINUX_VERSION-4.9 = .17
+LINUX_VERSION-4.9 = .18
 
 LINUX_KERNEL_HASH-3.18.43 = 1236e8123a6ce537d5029232560966feed054ae31776fe8481dd7d18cdd5492c
 LINUX_KERNEL_HASH-4.4.56 = 55181e24869e4b550ef756eecd1ab4f6774228c457f4830e9e779c6db4c2a2b6
-LINUX_KERNEL_HASH-4.9.17 = 9095c16550ebae9cbe0979a706116d95fbd8d270c1068857c4a8788de0f9ac7f
+LINUX_KERNEL_HASH-4.9.18 = a10eea12a85667636ea2503aebcdae3005c821333c489fc54e2a4e9fc44261d1
```
Comment (hnyman) [inline include/kernel-version.mk]: This file still refers to 4.9.18 although you have already changed the commit title for .19

**Comment by nicklowe** [comment]
There's a 4.9.20 now.

**Comment by Ansuel** [comment]
@hnyman thx that was just what i need... 
Will update to 4.9.20.... sorry for this mess

**Pull request closed without merging**